starknet_py/tests/unit/hash/transaction_test.py

```diff
@@ -129,7 +105,7 @@ def test_compute_invoke_transaction_hash(data, expected_hash):
                 "compiled_class_hash": 0x17B5169C770D0E49100AB0FC672A49CA90CC572F21F79A640B5227B19D3A447,
                 "account_deployment_data": [],
             },
-            0x7B31376D1C4F467242616530901E1B441149F1106EF765F202A50A6F917762B,
+            0x16081C54C3BEDC5079E0024896BFD85ED7E57FFD52B138CBC73AF0F34C7FCCE,
```

Comment:
note: Resource bounds affect the calculation of the transaction hash, hence its change.